docs/how-to-guides/how-to-setup-triggers.md

```diff
@@ -0,0 +1,101 @@
+---
+id: how-to-setup-triggers
+title: How to setup triggers
+sidebar_label: Triggers
+---
+
+Memgraph supports **database triggers** that can be executed if a particular type of event occurs. Events are related to changes in data, such as created, updated, and deleted nodes or relationships. 
+
+[![Related - Reference Guide](https://img.shields.io/static/v1?label=Related&message=Reference%20Guide&color=yellow&style=for-the-badge)](/reference-guide/triggers.md) 
+
+
+## How to create a trigger?
+
+You can create trigger by executing cypher commands. Creating a trigger will ensure that some procedural code is executed on certain type of event. 
+All triggers you will create are persisted on the disk, this means triggers will be active on database reruns and no information is lost.
+
+### Trigger execution upon node creation
+
+Node creation can be most common event your Memgraph database can react to. For example, you need to updated some value on created node. 
```

Comment:
Node creation is the most common event a Memgraph database can react to. For example, you can use it to update a certain value on an existing node/upon creating a node. ?
(sounds strange that it CAN be the most common event - it either is or it isn't)